Lib/test/test_smtplib.py

```diff
@@ -631,7 +631,7 @@ def testLineTooLong(self):
              'Mrs.C@somewhereesle.com':'Ruth C',
             }
 
-sim_auth = ('Mr.A@somewhere.com', 'somepassword')
+sim_auth = ('Mr.A@somewhere.com', 'somepassword密码')
```

Comment:
IMO we should keep the existing test case with an ASCII-only password, and add at least one more case with additional ones.

Reply:
E.g., add the following after `sim_auth = ...`, and then modify the test code to try them each in a loop (using `with self.subTest()` with an informative description).